src/middlewared/middlewared/plugins/idmap_/idmap_sss.py

```diff
@@ -0,0 +1,160 @@
+import pysss_nss_idmap as sssclient
```

Comment:
Do we need to expand our idmap CI tests?

Reply:
Once we add IPA join tests will be expanded. At this point we can't expand tests because we have to be properly joined to IPA (legacy LDAP bind won't cut it).